frontend/src/constants/coursesList.tsx

```diff
@@ -1,35 +1,102 @@
 const INITIAL_COURSES = [
```

Comment:
Do we need them at this moment? Seems like query already implemented

Reply:
In this branch we still need it in Courses page, where query isn't implemented